src/Components/Common/input/exemple.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import Input from './input';
+/**
+ *
+ * @param {object} state
+ * @returns {method} render
+ */
+class Form extends Component {
+  state = {};
+
+  /**
+   *
+   * @returns {jsx} react fragment
+   */
+  render() {
+    return (
+      <React.fragment>
+        <Input />
+        <Input />
+        <button className="btn btn-primary">submit</button>
+      </React.fragment>
+    );
+  }
+}
+
+export default Form;
```

Comment:
This example file is irrelevant, it is only needed in the development and should be removed after.